userChrome.css

```diff
@@ -46,3 +46,41 @@
 #reload-button {
 	margin-block-start: -2px !important;
 }
+
+
+/* context menuitem padding and rounded hover style changes */
+ menupopup > menuitem,
+ menupopup > menu
+  {
+     border-radius: 4px !important;
+     margin-left: 8px !important;
+     margin-right: 8px !important;
```

Comment:
To reduce as much as possible the line of code, always prefer shorthands (e.g `margin: 0 8px`) properties.

Reply:
Yes I tried it out with the css shorthands, but there seems to be a difference in the bottom margin in both cases. The one without the shorthands code is the default. Testing done on Windows 10, Firefox Nightly.

<img width="190" alt="no css shorthands" src="https://user-images.githubusercontent.com/56472130/122450758-6f1c7480-cfb8-11eb-9874-84d758758216.png">[Fig1:  margin: 0 8px not used. ]

<img width="194" alt="yes css shorthands" src="https://user-images.githubusercontent.com/56472130/122451012-befb3b80-cfb8-11eb-8140-289690938c29.png">[Fig2:  margin: 0 8px is used. ]

Hence I'm leaving it as `margin-left: 8px !important;` , `margin-right: 8px !important;`

Reply:
In that case, `margin: 0 8px [whatever you need to the bottom] 8px !important;` should works.

Reply:
Ahh ok. I'm setting it as `margin: 0 8px 2.5px 8px !important;`